scrapy/downloadermiddlewares/httpcompression.py

```diff
@@ -1,23 +1,33 @@
 import zlib
 
+
```

Comment:
good point, we could add flake8 style checks for PR. Flake8 could run on diff and point out all things like this.